### ISSUE TITLE
About the division of KITTI training dataset

### ISSUE THREAD

**Issue by Say2L** [Vegeta2020/SE-SSD#90]
Hello, I want to know how you divided the official KITTI training dataset when you get the results of KITTI test dataset?

**Comment by Vegeta2020** [comment]
Hi, we trained the model on the trainval set to get the results on the test set.

**Comment by Say2L** [comment]
Thanks for your reply.

**Comment by Tongjiaxun** [comment]
> Hi, we trained the model on the trainval set to get the results on the test set, which means we didn't divide but use the complete training dataset.

So, if no valid set, how to choose the best model?
